opencog/cython/opencog/atom.pyx

```diff
@@ -10,7 +10,7 @@ cdef class Atom(object):
         del self.handle
 
     def value(self):
-        return void_from_cptr(self.handle)
+        return self.handle.value()
```

Comment:
The `void_from_cptr(self.handle)` uniquely identifies an atom -- if the values are equal, then we are 100% certain that the atoms are identical.   By contrast, the hash has no such guarantee -- there is a good probability that inequivalent atoms will have different hashes, but its not that high -- the "birthday paradox" guarantees this. https://en.wikipedia.org/wiki/Birthday_problem  

What problem is this pull req trying to solve? The rest of the changes look OK; but the above breaks the semantics of "value" -- its always been guaranteed-unique, in the past.